essential-projects/data_model_contracts
# 🐛 fix getCollection-options-interface

## What did you change?

The `options`-parameter of the `getCollection`-method was of type `IPublicGetOptions`, when it should be of type `IPublicQueryOptions`. This branch fixes that.

## How can others test the changes?

Try using the `limit`-property in the `getCollection`-options and see how TypeScript is no longer shouting at you

## PR-Checklist

Please check the boxes in this list after submitting your PR:
